pkg/admissioncontroller/server/handlers/webhooks/utils.go

```diff
@@ -29,6 +29,13 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const (
+	// maxRequestBody is the maximum size of the `AdmissionReview` body.
+	// Same, fixed value from API server for general safety to reduce the odds of OOM issues while reading the body.
+	// https://github.com/kubernetes/kubernetes/blob/d8eac8df28e6b50cd0f5380e23fc57daaf92972e/staging/src/k8s.io/apiserver/pkg/server/config.go#L322
+	maxRequestBody = 3 * 1024 * 1024
```

Comment:
The problem in general is, that you can't be sure about the `--max-request-bytes` configuration of etcd backing the GAPI server. That's the main reason why for the moment I decided to choose the same value as K8s and gain some experience with it. In the worst case, GAPI returns an error because the object cannot be saved in etcd, so even if this value was too high, it wouldn't really harm or cause service outages.

Reply:
Ok, that sounds good. Thanks for explaining.